jsx/Form.js

```diff
@@ -630,6 +630,73 @@ var DateElement = React.createClass({
   }
 });
 
+/**
+ * Time Component
+ * React wrapper for a <input type="time"> element.
+ */
+var TimeElement = React.createClass({
```

Comment:
could this maybe be changed to ES6 syntax?

Reply:
I think that in this case the most important thing is internal consistency. The rest of the file is in ES5, so think that another PR should address ES6 syntax for the entirety of `Form.js` so that we are not mixing syntax standards.

Reply:
I think updating component by component as we go would be more efficient than issuing a PR to update the whole js file. Because components are not related to one another, i don't think it's important to keep consistency between them if the current standard is an old one.

the latest changes made to `Form.js` https://github.com/aces/Loris/pull/3559 is in ES6 so I think we should continue doing this

Reply:
With regard to #3559, I think that might have been a slight oversight. I had not seen that alteration to `Form.js` syntax, but I would have likely opposed it. A PR that refactors `Form.js` to meet ES6 can be made, but that's outside the scope of this PR which is specifically focused on providing a time element to the `Form.js` library file.